asserts/database.go

```diff
@@ -180,15 +184,19 @@ func OpenDatabase(cfg *DatabaseConfig) (*Database, error) {
 	dbCheckers := make([]Checker, len(checkers))
 	copy(dbCheckers, checkers)
 
+	dbNoAuthorityCheckers := make([]NoAuthorityChecker, len(noAuthorityCheckers))
+	copy(dbNoAuthorityCheckers, noAuthorityCheckers)
```

Comment:
AFAICT, the NoAuthorityChecker list is not configurable so I think the copying (to avoid mutation from outside, once the *Database is returned) is a bit pointless here.